lib/modules/manager/sbt/types.ts

```diff
@@ -1,11 +1,30 @@
+export type VariableContext = {
+  val: string;
+  sourceFile: string;
+  lineIndex: number;
+};
+type Variables = Record<string, VariableContext>;
```

Comment:
```suggestion
export type Variables = Record<string, VariableContext>;
```
always export types when they are used in other exported types


prefer interfaces